### PR TITLE
mac updates

### PR DESCRIPTION
**Pull request by notnoahpp**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "incremental": true,
+    "isolatedModules": false,
+    "jsx": "react-jsx",
+    "lib": ["esnext"],
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "noEmit": false,
+    "noErrorTruncation": false,
+    "noFallthroughCasesInSwitch": false,
+    "noImplicitAny": false, // never set this to false
```
Comment (notnoahpp) [inline tsconfig.json:48]: lol

Comment (notnoahpp) [reply tsconfig.json:48]: i think we should set this back to true and force use of `ts-ignore` or something like that

File: 0current/languages/javascript/examples/errors.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { log } from "./logit";
+
+class MyError extends Error {}
```
Comment (notnoahpp) [inline 0current/languages/javascript/examples/errors.ts:3]: you need to call super

File: linux/bash_cli_fns/mac.sh [hunk 3]
```diff
@@ -4,9 +4,14 @@ mac_shortcuts() {
...
+    vscode dupe line: start shift up/down
+    vscode jump head/tail: alt up/down
+    vscode move line: start up/down
+    vscode next file: alt start left/right
+    vscode sel multiline: alt start up/down
 	KEEP_FORGETTING
 }
 
-mac_cp_bashrc () {
+mac_cp_bashrc() {
```
Comment (notnoahpp) [inline linux/bash_cli_fns/mac.sh:15]: this isnt mac only

Comment (notnoahpp) [reply linux/bash_cli_fns/mac.sh:15]: should use the cp cmd that creates a backup

**Comment by noahehall** [comment]
TODO
- call super in js -> examples -> extends error
- put `mac_copybashrc` next to `sourcebashrc`

**Pull request merged**